src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/ProbationStatusDetailComparator.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.probation.courtcaseservice.service.model;
+
+import java.util.Comparator;
+
+import static java.util.Comparator.*;
+
+public class ProbationStatusDetailComparator {
```

Comment:
This looks much elegant using the language provision? 
public class ProbationStatusDetailComparator implements Comparator< ProbationStatusDetail> {
 ...
}

Reply:
Thanks,  On second thought, don't think we need a comparator at all.